routing/pathfind_test.go

```diff
@@ -1052,17 +1064,6 @@ func TestNewRoute(t *testing.T) {
 			expectedTotalAmount:   100130,
 			expectedTotalTimeLock: 6,
 			feeLimit:              noFeeLimit,
-		}, {
-			// Insufficient capacity in first channel when fees are added.
```

Comment:
Backwards routing removed the requirement for `newRoute` to check capacities. Before that, it could be that path finding returned an invalid route which must be caught in `newRoute`. That redundant check was still represent. I removed it together with removal of the channel capacity in the `Route` struct.